### ISSUE TITLE
nextcloud30Packages.app.contacts: version and sha256 do not match the tarball

### ISSUE THREAD

**Issue by kampka** [NixOS/nixpkgs#367892]
```
git checkout nixos-24.11
nix build .#nextcloud30Packages.apps.contacts
error: hash mismatch in fixed-output derivation '/nix/store/qr0kzgrhwy7dmnc5jwa9bzilndj0imki-contacts-v6.1.2.tar.gz.drv':
        likely URL: https://github.com/nextcloud-releases/contacts/releases/download/v6.1.2/contacts-v6.1.2.tar.gz
         specified: sha256-M3AC9KT3aMpDYeGgfqVWdI4Lngg/yw/36HSBS3N+G5c=
            got:    sha256-Slk10WZfUQGsYnruBR5APSiuBd3jh3WG1GIqKhTUdfU=
error: 1 dependencies of derivation '/nix/store/yznxxnzyjhq6h9inhlfjr1a7s5fzmcml-nextcloud-app-contacts-6.1.1.drv' failed to build
```

## Notify maintainers
@britter

**Comment by Atemu** [comment]
cc @onny

**Comment by bjornfor** [comment]
Fix backported to release-24.11 in a393ebcbea45dbf502a0a2979d124c574bbd5bd6.
